java/src/AdventDay1/SonarScanner_Day1.java

```diff
@@ -0,0 +1,33 @@
+package AdventDay1;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Scanner;
+
+public class SonarScanner_Day1 {
+
+    public static void main(String[] args) throws FileNotFoundException {
+
+        Scanner scanner = new Scanner(new FileReader("C:\\Users\\pedro\\IdeaProjects\\AdventOfCodeChallenges\\src\\AdventDay1\\input.txt"));
+        int[] depthDataArray = new int[2000];
```

Comment:
Here its better to use a relative path (`../../path/to/input.txt`) than an absolute one. Otherwise your code is only executable from your machine.

Reply:
Thanks, I made it work with the relative path now